tests/plugins/ajax/ajax.js

```diff
@@ -182,7 +182,7 @@
 			wait();
 		},
 
-		// (#1134)
+		// (#1134) (#4394)
```

Comment:
I wonder if it wouldn't be better to duplicate those tests. Ones with generic `load` (so stays as is) and other ones with specific type method. I know at the moment `load( ..., ..., type )` simply acts as a proxy for `loadType( ..., ... )` but if any more complex logic would appear in the future inside `load` itself we will have it covered and it will be better at detecting regressions. WDYT?

Reply:
Well, I just based on theoretical knowledge, that if we can cover some logic with some API - there is no need to test it again separately. It creates more tests to maintain. So if new complex logic appears in `load` we need to reflect it in more tests.

TBO - I know nothing about effective regression detection. But from my point of view - we probably catch it - because new API calls also fail for the same reasons why `load` might fail.

Reply:
Ok, makes sense for me :+1: